closed/OpenJ9.gmk

```diff
@@ -211,10 +212,6 @@ $(call openj9_add_jdk_shlibs, java.base, \
 	omrsig \
 	)
 
-ifneq (aarch64,$(OPENJDK_TARGET_CPU))
-  $(call openj9_add_jdk_shlibs, java.base, j9jit29)
```

Comment:
Isn't this line the one you want to do unconditionally now regardless of architecture?

Reply:
This line is replaced by adding j9jit29 to the list above.